src/WCSimRunAction.cc

```diff
@@ -103,6 +103,12 @@ void WCSimRunAction::BeginOfRunAction(const G4Run* /*aRun*/)
   //set detector & random options
   wcsimdetector->SaveOptionsToOutput(wcsimrootoptions);
   wcsimrandomparameters->SaveOptionsToOutput(wcsimrootoptions);
+
+  //set Tree for gathering photon info inside stepping action
+  photonTree = new TTree("photons","Photon Tree");
```

Comment:
This looks like a nice feature, but we need to be careful about bloating file sizes. Can you provide some file size information?
I'd suggest adding an boolean option to save this in `WCSimRunActionMessenger`

Reply:
Ok so I just run 10 events with OD.mac, and this is huge!
- Photon tree: 38 MB
- wcsimT tree: 1 MB

So we defintiely need to switch this off by default. I'd suggest something like 
- add a `bool` option to `WCSimRunActionMessenger` called `SaveWLSPhotonInfo`
- only create the `TTree` here if that's `true`
- only `Fill()` the tree in `WCSimSteppingAction.cc` if the tree is not `null`

It'd also be good to improve the description e.g.
```suggestion
  photonTree = new TTree("photons","Photons in WLS Tree");
```

Reply:
@tdealtry thanks for your suggestions but actually this feature is no longer needed. It was initially used to create the wavelength shifting process but we have now moved over to a completely geant4 implementation. As such I have removed this and other unused code in 78f458ebf883fc3a6652906b4315fa3c4cfcbaf7 .